sympy/printing/pretty/pretty_symbology.py

```diff
@@ -501,6 +501,7 @@ def xsym(sym):
                                 (U('DOUBLE-STRUCK CAPITAL N') +
                                  U('SUBSCRIPT ZERO'))),
     'Integers':                U('DOUBLE-STRUCK CAPITAL Z'),
+    'Rationals':               U('DOUBLE-STRUCK CAPITAL Q'),
```

Comment:
I don't see the pretty result here:
```
In [1]: S.Reals                                                                                                                   
Out[1]: ℝ

In [2]: S.Rationals                                                                                                               
Out[2]: Rationals
```

Reply:
I'm a neophyte at pretty printing. Any ideas?

Reply:
Something like this (although with a better ASCII fallback):
```diff
diff --git a/sympy/printing/pretty/pretty.py b/sympy/printing/pretty/pretty.py
index 6b9dd8c3f2..9c5802117c 100644
--- a/sympy/printing/pretty/pretty.py
+++ b/sympy/printing/pretty/pretty.py
@@ -172,6 +172,13 @@ def _print_Reals(self, e):
             inf_list = ['-oo', 'oo']
             return self._print_seq(inf_list, '(', ')')
 
+    def _print_Rationals(self, e):
+        if self._use_unicode:
+            return self._print_Atom(e)
+        else:
+            inf_list = ['-oo', 'oo']
+            return self._print_seq(inf_list, '(', ')')
+
     def _print_subfactorial(self, e):
         x = e.args[0]
         pform = self._print(x)
```
That gives:
```julia
In [1]: S.Rationals                                                                                                               
Out[1]: ℚ
```